docs/miscellaneous.rst

```diff
@@ -394,12 +394,14 @@ The following is the order of precedence for operators, listed in order of evalu
 +============+=====================================+============================================+
 | *1*        | Postfix increment and decrement     | ``++``, ``--``                             |
 +            +-------------------------------------+--------------------------------------------+
-|            | Function-like call                  | ``<func>(<args...>)``                      |
+|            | New expression                      | ``new <typename>``                         |
 +            +-------------------------------------+--------------------------------------------+
 |            | Array subscripting                  | ``<array>[<index>]``                       |
 +            +-------------------------------------+--------------------------------------------+
 |            | Member access                       | ``<object>.<member>``                      |
 +            +-------------------------------------+--------------------------------------------+
+|            | Function-like call                  | ``<func>(<args...>)``                      |
```

Comment:
I am hesitant to deviate too much from the specification of the AST, which treats `new <expression>` as the result of `new <typename>` composed with a function call or an index/member access. What's your take on this?